cachelib/allocator/CacheAllocator-inl.h

```diff
@@ -435,8 +434,11 @@ CacheAllocator<CacheTrait>::allocateInternalTier(TierId tid,
 
   // the allocation class in our memory allocator.
   const auto cid = allocator_[tid]->getAllocationClassId(pid, requiredSize);
-  util::RollingLatencyTracker rollTracker{
-      (*stats_.classAllocLatency)[tid][pid][cid]};
+
+  if (!fromBgThread) {
+    util::LatencyTracker tracker{stats().allocateLatency_};
```

Comment:
I don't think this will work correctly: as far as I understand, LatencyTracker measures times between it's creation and destruction, which in this case is between lines 439 and 441

Reply:
Made the fix by not tracking latency for batch size == 0